source/components/Dropdown/index.js

```diff
@@ -104,42 +133,77 @@ Dropdown.propTypes = {
      * React Component to display as the Dropdown Content
      */
     children: PropTypes.node,
+
     /**
-     * Whether or not dropdown should have a slight drop shadow
+     * Should dropdown content be scrollable
      */
     contentScrollable: PropTypes.bool,
+
     /**
-     * Hides chevron in dropdown toggle
+     * Should dropdown content be left-aligned with the dropdown toggle
      */
     dropdownLeftAligned: PropTypes.bool,
```

Comment:
Consider using `align: PropTypes.oneOf(['left', 'right'])` instead of two different props.

Reply:
This is just to preserve some sort of compliance between design system components in react and ember.

Reply:
I'd love to do an audit in near future, to make sure that APIs of both DS are in sync. I think if you learn one - you should be able to apply the same knowledge to use the other DS implementation.